src/Orleans/Runtime/OutsideRuntimeClient.cs

```diff
@@ -27,6 +29,7 @@ internal class OutsideRuntimeClient : IRuntimeClient, IDisposable, IClusterConne
         private Logger callBackDataLogger;
         private ILogger timerLogger;
         private ClientConfiguration config;
+        private ClientMessagingOptions clientMessagingOptions;
```

Comment:
I don't see any usages of this other than the assignment. What is it for?

Reply:
we've references for "config" other places, if more usages will be migrated we'll could that variable, if not, we can remove it later of course, like I removed the unused config members from one of the classes.